commands.go

```diff
@@ -2256,6 +2310,18 @@ func (c cmdable) ZUnionStore(ctx context.Context, dest string, store *ZStore) *I
 	return cmd
 }
 
+func (c cmdable) ZRandMember(ctx context.Context, key string, count int, withScores bool) *StringSliceCmd {
+	args := make([]interface{}, 0, 4)
+	args = append(args, "zrandmember", key, count)
```

Comment:
Is it okay to use count=0 here too? If so having the comment here wouldn't hurt :)